README.md

```diff
@@ -1,30 +1,34 @@
 # Licenser
-An extension for adding license headers and LICENSE file in your workspace.
+
+An extension for adding license headers and LICENSE files into your workspace.
 You can find this extension on [Marketplace for VS Code](https://marketplace.visualstudio.com/items?itemName=ymotongpoo.licenser).
 
 ## Install
 
-Launch your VS Code, open Quick Open box (Mod+P), and run following command.
+Launch VS Code, go to the *Quick Open* menu (`Ctrl+P` on Windows/Linux, `⌘P` on OS X), and type:
```

Comment:
Thanks for the catch. Can you change "OS X" to "macOS" to be more accurate?

Reply:
Yep! I just pushed another commit that does this.